qiskit/pulse/schedule.py

```diff
@@ -50,13 +50,20 @@ class Schedule(abc.ABC):
     def __init__(self,
                  *schedules: Union[Union['Schedule', Instruction],
                                    Tuple[int, Union['Schedule', Instruction]]],
-                 name: Optional[str] = None):
+                 name: Optional[str] = None,
+                 metadata: Optional[dict] = None):
```

Comment:
Thank you, this is great and can be used to transmit circuit header information to pulse assembly.

Reply:
Oh, that reminded me that I should have added the circuit metadata to the output of the pulse scheduler too. Right now it's not preserving it. I'll do that now